src/yoroi-wallets/utils/utils.ts

```diff
@@ -98,9 +116,31 @@ export const Quantities = {
   denominated: (quantity: Quantity, denomination: number) => {
     return Quantities.quotient(quantity, `${10 ** denomination}`)
   },
+  atomic: (data: Quantity | BigNumber | string | number, denomination: number) => {
+    const stripped = data.toString().replace(/[^0-9.-]/g, '')
+    const value = (stripped.length > 0 && new BigNumber(stripped).isZero() !== true ? stripped : '0') as Quantity
+    return new BigNumber(value).toFixed(denomination).toString().replace(/[.,]/g, '') as Quantity
+  },
+  zero: () => '0' as Quantity,
```

Comment:
why is this necessary? why is it a function?

Reply:
🤦